valhalla/proc/request_builder.py

```diff
@@ -103,7 +103,7 @@ def get_costing_options(costing_options, profile, mode):
         for cost in costing_options:
             if cost[1]:
                 if cost[0] in [TRUCK_COSTING.WIDTH, TRUCK_COSTING.LENGTH]:  # Temp bcs of Valhalla issue
-                    params[profile][cost[0]] = round(cost[1] / 3.28084, 2)
+                    params[profile][cost[0]] = cost[1]
```

Comment:
again no need to keep IF for width and length so remove it completly:
```python
if cost[1]:
   params[profile][cost[0]] = cost[1]
```

Reply:
Same problem.

It should be fixed now.

Thank you very much for the feedback.